packages/html-program-viewer/test/smoke-test.ts

```diff
@@ -0,0 +1,14 @@
+import { BasicTestRunner } from "@cadl-lang/compiler/testing";
+import { createViewerTestRunner } from "./test-host.js";
+
+describe("Smoke tests", () => {
+  let runner: BasicTestRunner;
```

Comment:
is there a mechanism here for accessing the contents of the output html.  It would be nice to test for specific language element renderings, if only for regression protection

Reply:
ultimately we should probably have unit test for all the rendered component, there is many react libraries that will let us do that. I guess here could add a basic snapshot test to make sure the generated html doesn't change